rdr_service/model/questionnaire_response.py

```diff
@@ -42,6 +42,14 @@ class QuestionnaireResponse(Base):
     """The actual time the participant completed the questionnaire"""
     language = Column("language", String(2), nullable=True)
     """Language that the response was completed in"""
+
+    # Can be used to indicate equality between sets of answers
+    answerHash = Column('answer_hash', String(32), nullable=True)
+    """@rdr_dictionary_internal_column"""
+
+    isDuplicate = Column("is_duplicate", Boolean, server_default=expression.false())
+    """Indicates that the response is a duplicate of another that was previously received and should be ignored"""
```

Comment:
Should we add these for every addition so the data dict will pick up the declaration/explanation?

Reply:
Yeah! Ideally we'll incrementally get all the fields documented, but any new fields that are added should have documentation. That'll help us and other teams understand what it is and how to use it.